docs/daemon/cli/kwild_cli.mdx

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 1
+sidebar_label: "Kwild CLI"
+id: daemon-cmdline
+title: Kwild Command Line Interface Guide
+description: Kwild Command Line
+slug: /daemon/cmdline
+---
+
+The kwild command line interface (CLI) provides a comprehensive suite of commands and utilities for configuring and interacting with Kwild services
```

Comment:
It seems you switch off between "kwild" and "Kwild" interchangeably.  We should use `kwild` and "Kwil daemon"